usng.js

```diff
@@ -382,6 +462,166 @@
             return result;
         },
 
+        serializeUPS(upsCoordinates = {}) {
+            //verify that the passed in coords is a valid ups object
+            try { this.UPStoLL(upsCoordinates) } 
+            catch (err) { throw err }
```

Comment:
✏️ I don't believe I saw any tests checking for error conditions?

Reply:
You should now see unit tests for those error conditions.